TestCases/sliding_interface/uniform_flow/uniform_SM.cfg

```diff
@@ -0,0 +1,267 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Agree, I added a few more, including the one with 5 zones which I believe is critical.
I was thinking that, right now, parallel regression works using only 2 processes for each test.
We should have some of them testing the code using also a different number of threads, as some bugs may not arise with just 2.